go/pkg/app/app.go

```diff
@@ -89,8 +89,6 @@ func init() {
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	utilruntime.Must(v1alpha2.AddToScheme(scheme))
 	// +kubebuilder:scaffold:scheme
-
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
This was introduced in https://github.com/kagent-dev/kagent/commit/2cb72c310970678afc4a3b002388906798344bcc - although I can't see any reason why it's needed.